packages/manager/modules/pci/src/components/project/images-list/images-list.controller.js

```diff
@@ -42,11 +43,13 @@ export default class ImagesListController {
   }
 
   getImages() {
-    return this.PciProjectImages.getImages(this.serviceName).then((images) => {
-      this.images = images;
-      this.updateImages(images);
-      return images;
-    });
+    return this.PciProjectImages.getImages(this.serviceName)
+      .then((images) => orderBy(images, ['name'], ['desc']))
```

Comment:
you PR says 'newest' but you are sorting by name

Reply:
You can't sort by `name` to sort by `date`.

Reply:
The aims is to display the new OS images first.
Eg for Fedora: `['Fedora 33', 'Fedora 32']`

Reply:
is't an issue @cbourgois ?

Reply:
This will work: 

```js
_.orderBy(
    [
        {
            name: 'Fedora 33'
        },
        {
            name: 'Fedora 34'
        },
        {
            name: 'Fedora 32'
        }
    ],
    ['name'], 
    ['desc'],
)
```

will output 

```
Fedora 34 
Fedora 33 
Fedora 32
```

But not this:

```js
_.orderBy(
    [
        {
            name: 'Fedora 1'
        },
        {
            name: 'Fedora 10'
        },
        {
            name: 'Fedora 9'
        }
    ],
    ['name'], 
    ['desc'],
)
```

will output

```
Fedora 9
Fedora 10
Fedora 1
```

So yes it's an issue.

Reply:
Outdate.